Symfony/CS/Tests/Fixer/PSR2/UnneededControlParenthesesFixerTest.php

```diff
@@ -0,0 +1,363 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\PSR2;
+
+use Symfony\CS\Fixer\PSR2\UnneededControlParenthesesFixer;
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ *
+ * @internal
+ */
+final class UnneededControlParenthesesFixerTest extends AbstractFixerTestBase
+{
+    public function testConfigure()
+    {
+        $loopsProperty = new \ReflectionProperty('\Symfony\CS\Fixer\PSR2\UnneededControlParenthesesFixer', 'loops');
+        $loopsProperty->setAccessible(true);
+        $loops = $loopsProperty->getValue(null);
+        UnneededControlParenthesesFixer::configure(array('return', 'switch_case'));
+
+        $expected = '<?php
+            echo ("foo");
+            case true;
+            return 42;
+        ';
+
+        $input = '<?php
+            echo ("foo");
+            case (true);
+            return (42);
+        ';
+
+        $this->makeTest($expected, $input);
+
+        UnneededControlParenthesesFixer::configure(array());
+        $loopsProperty->setValue($loops);
+    }
+
+    /**
+     * @dataProvider provideFixCases
+     * @requires PHP 5.5
+     */
+    public function testFix($expected, $input = null)
```

Comment:
Test sample where all options are active: github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/Symfony/CS/Tests/Fixer/Contrib/PhpUnitConstructFixerTest.php#L30-L35

Test sample where all options are inactive: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/Symfony/CS/Tests/Fixer/Contrib/PhpUnitConstructFixerTest.php#L38-L43

Test sample with only one option active (separately for each of them): https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/Symfony/CS/Tests/Fixer/Contrib/PhpUnitConstructFixerTest.php#L47-L53